DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/AccessoAllaPiattaforma.tex

```diff
@@ -1,58 +1,290 @@
 \UC{Registrazione}
+\begin{figure}[H]
+	\centering
+	\includegraphics[scale=0.6]{Immagini/DiagrammiUC/UC1.1Registrazione}
+	\caption{Diagramma di \actualUC: Registrazione}
+	\label{fig:RegistrazionePiattaforma}
+\end{figure}
+
+L'utente non autenticato che non dispone di credenziali può registrarsi e accedere come acquirente nella piattaforma compilando alcuni campi obbligatori per poter procedere.
+\begin{itemize}
+	\item \textbf{Attori primari:} utente non autenticato;
+	\item \textbf{Attori secondari:} \glo{identity manager};
+	\item \textbf{Precondizione:} l'utente non è ancora presente nella piattaforma;
+	\item \textbf{Postcondizione:} l'utente è registrato con un account acquirente ed è autenticato come tale sulla piattaforma;
+	\item \textbf{Scenario principale:}
+	\begin{itemize}
+		\item L'utente accede al sistema;
+		\item L'utente non autenticato si trova nella schermata di registrazione;
+		\item (UC) - Inserisce tutti i dati per compilare il modulo di registrazione;
+		\item L'utente non autenticato conferma il proprio indirizzo email inserito;
+		\item L'utente possiede un account presso il sistema.
+	\end{itemize}
+	\item \textbf{Estensioni:}
+	\begin{enumerate}
+		\item L'utente inserisce un indirizzo email e l'identity manager segnala che è già presente nella piattaforma
+		\begin{itemize}
+			\item L'utente non viene registrato presso il sistema;
+			\item (UC) - Visualizzazione messaggio di errore in caso di email già registrata nella piattaforma;
+			\item Viene fornita all'utente la possibilità di modificare il proprio indirizzo email inserito.
+		\end{itemize}
+		\item L'utente compila i campi dati per la password e la conferma della stessa con due password diverse, in questo caso:
+		\begin{itemize}
+			\item L'utente non viene registrato presso il sistema;
+			\item L'identity manager segnala che le due password inserite non coincidono;
+			\item (UC) - Visualizzazione messaggio di errore in caso di password e password di conferma diverse;
+			\item L'utente può modificare le password inserite.
+		\end{itemize}
+		%\item l'utente prova ad accedere alla piattaforma con i dati inseriti nel modulo senza aver confermato il proprio indirizzo email
+		%\begin{itemize}
+		%	\item (UC) - Visualizzazione messaggio di errore in caso di credenziali sconosciute;
+		%	\item L'utente non possiede un account presso il sistema.
+		%\end{itemize}
+	\end{enumerate}
```

Comment:
Io cambierei questa cosa qua mettendo la parte testuale dentro uno scenario alternativo e terrei il punto estensioni come una lista di estensioni e basta, se no dovremmo cambiare tutto il documento per uniformarlo e scriverlo in questo modo